tvb/analyzers/fft.py

```diff
@@ -110,7 +110,7 @@ def evaluate(self):
             time_series = numpy.concatenate(segments, axis=4)
         else:
             self.segment_length = time_series_length
-            time_series = self.time_series.data[:, :, :, numpy.newaxis]
+            time_series = self.time_series.data[:, :, :, :, numpy.newaxis]
```

Comment:
Shouldn't we have a similar change at line 109 above ?

Reply:
This change does not make sense given the other changes in this PR.

Reply:
The pull request covers 2 issues: TVB-1837 and bug TVB-2282.
This line is the only change related to TVB-2282